misc/loop/docker-compose.production.yml

```
@@ -0,0 +1,123 @@
+version: "3"
+
+networks:
+  portal-loop:
+    name: portal-loop
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.177.0.0/16
+
+services:
+  traefik:
+    image: "traefik:v2.10"
+    restart: unless-stopped
+    command:
+      - "--api.insecure=true"
+      - "--providers.file=true"
+      - "--providers.file.watch=true"
+      - "--providers.file.directory=/etc/traefik/configs"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.private.address=:81"
+      - "--entrypoints.web.http.redirections.entrypoint.to=websecure"
+      - "--entrypoints.web.http.redirections.entrypoint.scheme=https"
+      - "--entrypoints.web.http.redirections.entrypoint.permanent=true"
+      - "--entryPoints.web.forwardedHeaders.insecure"
+      - "--entrypoints.traefik.address=:8080"
+
+      - "--entrypoints.websecure.address=:443"
+      # - "--certificatesresolvers.le.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory"
+      - "--certificatesresolvers.le.acme.email=dev@gno.land"
+      - "--certificatesresolvers.le.acme.storage=/letsencrypt/acme.json"
+    networks:
+      - portal-loop
+    ports:
+      # useless because of network_mode: host
+      # just here for readability
+      - "80:80"
+      - "443:443"
+      - "8080:8080"
+    volumes:
+      - ./traefik:/etc/traefik/configs
+      - ./traefik/letsencrypt:/letsencrypt
+
+  gnoweb:
+    image: ghcr.io/gnolang/gno/gnoweb-slim
+    restart: unless-stopped
+    entrypoint:
+      - gnoweb
+      - --bind=0.0.0.0:8888
+      - --remote=traefik:81
+      # - --faucet-url="http://faucet.portal.gno.local:81"
+      # - --faucet-url
+      # - "http://localhost:9000"
+      - --help-chainid
+      - portal-loop
+      - --help-remote
+      - https://rpc.portal.gno.land:443
```

```suggestion
      - https://rpc.portal.gnoteam.com:443
```

This is the current good value, which helps generate valid commands for the help page.